### DISCUSSION TITLE
A better UI

### DISCUSSION THREAD

**Discussion by fabiencharrasse** [fisharebest/webtrees#5045] [Ideas]
Hello.

I started my genealogy adventure with filae.com and I have to say that the user experience to build its tree is particularly easy and well thought out.

Overall on webtrees, a lot more of clicks/ pages reload are needed and the whole is not very "visual". This does not remove the fact that this is a great software but I think the user experience could be improved.

Here are some screenshots to illustrate

![image](https://github.com/user-attachments/assets/3403863c-e63b-4e77-9bd0-1956e5b834a1)

You can zoom / unzoom the tree, move with drag&drop, focus on a person of the tree (to display childs / brother / sisters, etc => when you focus, there's a page reload, url become : https://en.filae.com/v4/genealogie/tree.mvc/familytree?personId=XXXXXXX). All editing is done with popin. Up to 6 generations are displayed and you can click on the grey arrow in the circle (bottom) to display next generations.

![image](https://github.com/user-attachments/assets/a1d8b147-9518-4831-8532-6e074f9532d8)

![image](https://github.com/user-attachments/assets/d54c759c-c6b9-410c-a79b-a41c815d1a96)

Hope it help and give some nice ideas

**Comment by hartenthaler** [comment]
Have you seen the custom module GVExport? You can visualise your tree, zoom, move, add partners, parents and children.
